### PR TITLE
Add marquez-airflow-quickstart

### PR DESCRIPTION
**Pull request by wslulciuc**
This PR adds the `marquez-airflow-quickstart` module to `marquez` under the dir `examples/airflow`, see #826

### CODE REVIEW

**Review by phixMe — APPROVED** [review]
Looks good @wslulciuc

**Review by julienledem — APPROVED** [review]
looking good. One minor comment

**Comment by julienledem** [comment]
Ah right, this is an import. Just push to main. (but one of the two branches at a time)

File: examples/airflow/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM puckel/docker-airflow:1.10.6
+
+LABEL maintainer "WeWork<data-platform@wework.com>"
```
[the remark this answers is not in the document]
Comment (wslulciuc) [reply examples/airflow/Dockerfile:3]: Nope! Good catch 👍

**Pull request merged**